docs/openapi/api.yaml

```diff
@@ -105,6 +105,20 @@ paths:
     $ref: './audit-api.yaml#/latest-audit-for-site'
   /sites/{siteId}/{auditType}:
     $ref: './audit-api.yaml#/update-handler-type-config-for-site'
+  /sites/{siteId}/opportunities:
+    $ref: './site-opportunities.yaml#/site-opportunities'
+  /sites/{siteId}/opportunities/by-status/{status}:
+    $ref: './site-opportunities.yaml#/site-opportunities-by-status'
+  /sites/{siteId}/opportunities/{opportunityId}:
+    $ref: './site-opportunities.yaml#/site-opportunity'
+  /sites/{siteId}/opportunities/{opportunityId}/suggestions:
+    $ref: './site-opportunities.yaml#/site-opportunity-suggestions'
+  /sites/{siteId}/opportunities/{opportunityId}/suggestions/by-status/{status}:
+    $ref: './site-opportunities.yaml#/site-opportunity-suggestions-by-status'
+  /sites/{siteId}/opportunities/{opportunityId}/suggestions/{suggestionId}:
+    $ref: './site-opportunities.yaml#/site-opportunity-suggestion'
+  /sites/{siteId}/opportunities/{opportunityId}/suggestions/status:
```

Comment:
what is the difference between
`/sites/{siteId}/opportunities/{opportunityId}/suggestions/by-status/{status}:`
vs
`/sites/{siteId}/opportunities/{opportunityId}/suggestions/status:`

Reply:
`GET /sites/{siteId}/opportunities/{opportunityId}/suggestions/by-status/{status}` lists suggestions by status
`PATCH /sites/{siteId}/opportunities/{opportunityId}/suggestions/status` updates the status of one or more suggestions

Reply:
wouldn't it make sense to use the same path for both GET and PUT/PATCH?

Reply:
Then I'd leave the `GET /sites/{siteId}/opportunities/{opportunityId}/suggestions/by-status/{status}` as is, because it aligns with other get list of filtered entities APIs.
And turn `PATCH /sites/{siteId}/opportunities/{opportunityId}/suggestions/status` into `PATCH /sites/{siteId}/opportunities/{opportunityId}/suggestions` with the only property allowed to update for now being status.
@solaris007 wdyt?

Reply:
that is confusing - previously the URL made clear that we're updating status. like this we're obfuscating? i'd leave it as it is.